### ISSUE TITLE
default value for parameters then they are actually needed

### ISSUE THREAD

**Issue by gonzaponte** [next-exp/IC#271]
In `peak_functions_c.py` we have:
````
cpdef find_S12(double [:] csum,  int [:] index,
    time=(), length=(),
    int stride=4, rebin=False, rebin_stride=40):
````
The `time`and `length` arguments have a default empty tuple value, but they actually need to be a length-2 tuple, or the code will crash. Is there any reason for them to have default values?

**Comment by jjgomezcadenas** [comment]
I believe there is no real reason for these parameters to take defaults, you are right. Let me look into it.

**Comment by jjgomezcadenas** [comment]
Fixed now, will be included in the upcoming PR

**Comment by jjgomezcadenas** [comment]
closed through #265
